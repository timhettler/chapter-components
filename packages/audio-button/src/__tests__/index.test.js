import React from 'react';
import Component from '../';
import renderer from 'react-test-renderer';

it(`${Component.name} renders correctly when muted`, () => {
  const tree = renderer.create(<Component isMuted={true} />).toJSON();
  expect(tree).toMatchSnapshot();
});

it(`${Component.name} renders correctly when unmuted`, () => {
  const tree = renderer.create(<Component isMuted={false} />).toJSON();
  expect(tree).toMatchSnapshot();
});
