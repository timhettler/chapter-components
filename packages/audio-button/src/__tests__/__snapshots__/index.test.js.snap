// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioButton renders correctly when muted 1`] = `
<button
  className="btn btn--is-dark"
>
  <span
    className="btn__inner"
  >
    <svg
      className="icon_style"
      height="100%"
      viewBox="0 0 20 20"
      width="100%"
    >
      <path
        d="M15 8.59l-2.12-2.13-1.42 1.42L13.6 10l-2.13 2.12 1.42 1.42L15 11.4l2.12 2.13 1.42-1.42L16.4 10l2.13-2.12-1.42-1.42L15 8.6zM4 7H0v6h4l5 5V2L4 7z"
      />
    </svg>
  </span>
</button>
`;

exports[`AudioButton renders correctly when unmuted 1`] = `
<button
  className="btn btn--is-dark"
>
  <span
    className="btn__inner"
  >
    <svg
      className="icon_style"
      height="100%"
      viewBox="0 0 64 64"
      width="100%"
    >
      <path
        d="M33.1 43.82c10.18-11.38 1.06-22.43 0-23.66l6.1-5.42c5.33 6 12.2 20.87 0 34.52zm12.51 12.25c21.28-23.8.88-47.16 0-48.14l6.44-5.75c.27.3 26.36 30.16 0 59.64z"
      />
      <path
        d="M0 32V21.54h7.86L23.83 5.56v52.88L7.86 42.46H0V32"
      />
    </svg>
  </span>
</button>
`;
